src/types/itemTypes.ts

```diff
@@ -20,16 +20,14 @@ export interface SearchResultsItem {
 export interface ItemLocation extends JSONLDValue {
   customerCode?: string
   prefLabel?: string
-  endpoint?: ItemLocationEndpoint
+  endpoint?: string
 }
 
 export type ItemLocationKey = "Schwarzman" | "Performing" | "Schomburg"
 
-export type ItemLocationEndpoint = "schwarzman" | "lpa" | "schomburg"
```

Comment:
Why was this removed?

Reply:
Hmm I see it was to reduce the type casting but it makes it clear what the values are.

Reply:
I agree that it's good to have them clearly defined but thought it was a bit of a headache to have to add it on every fixture and elsewhere. The values are still enforced in the locationEndpointsMap as well as the key type (ItemLocationEndpoint) but if you'd rather have the values for the endpoints type enforced I can add that back in.

Reply:
Can leave out for now